environments/site/hooks/pre.yml

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
I think this is the case, but just to confirm @oneswig did this cause the config to be baked into the new image? 

Just because this hook won't be run in staging/production without also being added as a hook in the environments.